src/templates/_common/components/title.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 const Title = () => (
-    <title>{`${it.title ? `${it.L(it.title)} | ` : ''}${'Online trading platform | Deriv'}`}</title>
+    <title>{`${it.title ? `${it.L(it.title)} | ` : ''}${`${it.L('Online trading platform')} | ${it.L('Deriv')}`}`}</title>
```

Comment:
`${it.L('Deriv')}` is this needed?

Reply:
since deriv is a name, there is no need for translation. you're right :pray: